python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -105,8 +105,9 @@ def parsePipelineArgs():
     $ python ap_pipe.py input_location -t template_location -o output_location -i "visit=12345 ccdnum=5"
```

Comment:
Please update this usage example so it is consistent with the accepted arguments; we must have missed this when template handling was changed.

Reply:
Or maybe it is consistent, it's just difficult to show all the possible combinations of use cases; my mistake.

Reply:
You mean adding `--calib`, I assume? I can do that.

On the other hand, would you be ok with removing this line from the description? The output looks kind of weird:

    $ python bin/ap_pipe.py -h
    usage: ap_pipe.py [-h] [-o OUTPUT] [-i DATAID] [--no-skip]
                      [--templateId TEMPLATEID | -t TEMPLATEREPO]
                      input

    Process raw decam images with MasterCals from basic processing --> source association

    USAGE:
    $ python ap_pipe.py input_location -t template_location -o output_location -i "visit=12345 ccdnum=5"

Reply:
Goes to show how often I try running things with the `-h` flag! Sure, it's a bit redundant with the actual usage info that prints out 👍